CHANGELOG.md

```diff
@@ -307,11 +307,11 @@ Additionally, Postgres [upgrading](docs/maintenance-postgres.md#upgrading-postgr
 
 # 2019-10-04
 
-## Postgres 12 support
+## Postgres 13 support
```

Comment:
This should probably be a new entry rather than editing the Postgres 12 entry from last year

Reply:
Whoops, should have paid more attention to what I was editing. I'll leave the new changelog entry to you.